src/test/kotlin/com/igorwojda/list/maxsublistsum/solution.kt

```diff
@@ -84,3 +84,11 @@ private object Solution4 {
             .maxOrNull()
     }
 }
+
+private object Solution5 {
+    private fun maxSubListSum(list: List<Int>, numElements: Int) = list.windowed(numElements)
+        .toMutableList()
+        .maxOfOrNull {
```

Comment:
Lambda containing one instruction should be formatted in one line:

```koltin
.maxOfOrNull { it.sum() }
.toMutableList()
```

Pleas update the code